apis/oauth/src/main/java/org/jclouds/oauth/v2/config/CertificateFingerprintSupplier.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http:://www.apache.org/licenses/LICENSE-2.0
```

Comment:
The build is failing because an invalid license. There is a double colon here `::` :)